lxd/patches.go

```diff
@@ -413,27 +411,13 @@ func patchDBNodesAutoInc(name string, d *Daemon) error {
 			return nil // Nothing to do.
 		}
 
-		// Only apply patch on leader, otherwise wait for it to be applied.
-		var localConfig *node.Config
-		err = d.db.Node.Transaction(context.TODO(), func(ctx context.Context, tx *db.NodeTx) error {
-			localConfig, err = node.ConfigLoad(ctx, tx)
-			return err
-		})
-		if err != nil {
-			return err
-		}
-
-		leaderAddress, err := d.gateway.LeaderAddress()
+		isLeader, _, err := d.leaderInfo()
 		if err != nil {
-			if errors.Is(err, cluster.ErrNodeIsNotClustered) {
-				break // Apply change on standalone node.
-			}
-
 			return err
 		}
 
-		if localConfig.ClusterAddress() == leaderAddress {
-			break // Apply change on leader node.
+		if !isLeader {
```

Comment:
```suggestion
		if isLeader {
```

Could this perhaps be the correct form?

Reply:
Yep that one was wrong! Thanks :)